src/layouts/articles/article-3/extra/comment-list.component.tsx

```diff
@@ -1,10 +1,9 @@
-import React from 'react';
+import React, { ReactElement } from 'react';
```

Comment:
`ReactElement` can be used by only importing a `React` package without any need to use destruction.
Let's use that as a rule in the entire code base.

```suggestion
import React from 'react';
```